shared/message/platform/gazebo/RawSensors.proto

```diff
@@ -23,8 +23,6 @@ package message.platform.gazebo;
 import "message/platform/darwin/DarwinSensors.proto";
 
 message RawSensors {
-    // The name of the model in the simulation that these sensors are from
-    string model = 1;
-    // The sensors object for this model
+    string                        model   = 1;
```

Comment:
I think these comments should be added back in, ppl might not know what 'model' is otherwise. I wasn't sure what it meant.